system/cmd_runner_interface.go

```diff
@@ -19,6 +19,9 @@ type Command struct {
 	// Don't echo stdout/stderr
 	Quiet bool
 
+	// If invoking a shell script (for platform-specific behaviors)
+	IsScript bool
```

Comment:
Why add `IsScript`?  Shouldn't handling PowerShell scrips (e.g. `C:\foo.ps1`) vs. cmdlets (e.g. `Import-Certificate -FilePath C:\cert.crt`) be the responsibility of the user as they **must** be handled differently?  Additionally, this is a no-op on *Nix.